ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+- PCSX2 version:
+``(For example, stable build v1.4.0 (or) development builds v1.5.xxx)``
+
+
+- PCSX2 options:
+``(For example, Clamping/Rounding, Non-default settings (or) speedhacks used)``
+
+
+- Plugins used:
+``(Relevant/Non-default settings of GS/PAD/SPU2/CDVD/USB plugins )``
+
+
+- Description of the issue:
+``(What's the problem?)``
+
+
+- How to replicate:
+``(Step by step guide to replicate the issue?)``
+
+
+- Last know version to work:
+``(If the issue was a regression)``
```

Comment:
I'd add "preferably not much earlier than the version on which the issue is reported"